.circleci/config.yml

```diff
@@ -70,6 +70,8 @@ jobs:
           command: << parameters.lein_test_command >>
       - run:
           command: .circleci/e2e.sh
+      - run:
+          command: lein with-profile -user,+test eastwood
```

Comment:
Good to self-lint. It's not very formatting-stacky to lint all files, but it's a good idea (self-reference tends to be logically weak)

Reply:
I think this is a good change, but i'd rather implement it in terms of formatting-stack. I did some work to support cli invocation.

But i'd be fine with replacing this CI step when that's ready.

Reply:
I'd suggest having both for the hinted reason: using f-s to lint f-s is somewhat weak. It can self-report being OK while it's not.

One would run the self-linting before Eastwood though, for fast feedback.

This could be an entire sequence:

* f-s self-lint (kondo + eastwood, git diff only)
* clj-kondo, whole project
* eastwood, whole project

Maybe a little annoying for our standards of fast CI linting, but it could catch an issue the test suite didn't

Reply:
In my suggestion it'd run the linter across the entire project as well. But we'll cross that bridge when we get there.

Reply:
There could be an issue in strategies, in reporting, in interaction with a given linter impl... bunch of things that can go wrong